dandiapi/api/asset_paths.py

```diff
@@ -240,7 +240,7 @@ def update_asset_paths(old_asset: Asset, new_asset: Asset, version: Version):
 @transaction.atomic
 def add_version_asset_paths(version: Version):
     """Add every asset from a version."""
-    print('\t Leaves...')
+    # Leaves
```

Comment:
@AlmightyYakob is there value in printing this out? Should we replace this with a `logger` call?

Reply:
I don't know. I'm happy to change it to a `debug` or `info` log if you want.

Reply:
I'm kind of surprised those print statements are still there. I think they were originally added when the asset paths stuff was new, to make sure things were working correctly. You can remove them.